toadr3/iso_date.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+import re
+
+
+def parse_iso8601_duration(duration: str) -> datetime.timedelta:
```

Comment:
How wierd that this is not present in stdlib!

But it would have been [solved](https://stackoverflow.com/a/77957966/2140093) if we used pydantic:-p

Reply:
🤪